app/forms/update_user_email_form.rb

```diff
@@ -15,7 +15,7 @@ def initialize(user)
   end
 
   def submit(params)
-    email = params[:email].downcase
+    email = params[:email].downcase.strip
```

Comment:
ah ok, did not see that part (hard to see a single whitespace!)

I think the fact that I overlooked that might be further evidence that a small utility class that formats these strings might be helpful. That way, don't need to test the `strip.downcase` behavior everywhere, just need to test it once and then ensure that we are calling the utility class where needed.